plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/protobuf/TestingUnsupportedNestedTypes.java

```diff
@@ -0,0 +1,3204 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Please provide some readme file, to explain how to generate this file. The best would be to have the shell script.